Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 let package = Package(
     name: "DeckUI",
-    platforms: [.macOS(.v12), .iOS(.v14)],
+    platforms: [.macOS(.v12), .iOS(.v15)],
```

Comment:
Updated because of the task API used in the Media.Kind.bundleVideo.
I believe it can be rewritten with just `onAppear`. What do you think?

Reply:
Oh, I'm fine with 15 😊